ethereum/contracts/utils/MerkleProof.sol

```diff
@@ -61,4 +61,53 @@ library MerkleProof {
 
         return computedHash == root;
     }
+
+    function computeMerkleLeafAtPosition(
```

Comment:
@philipstanislaus i presume we still need the position to compute the root?

if so, can we just use this code below but revert instead of "return false;" ?

Reply:
Yes, definitely. Fixed just two minor issues. Also should name this function: 

```suggestion
    function computeRootFromProofAtPosition(
```

Reply:
Can we use this function in the `verifyMerkleLeafAtPosition` above?